var/spack/repos/spack_repo/builtin/packages/petsc/package.py

```diff
@@ -269,6 +269,10 @@ class Petsc(Package, CudaPackage, ROCmPackage):
     variant("hwloc", default=False, description="Activates support for hwloc")
     variant("kokkos", default=False, description="Activates support for kokkos and kokkos-kernels")
     variant("fortran", default=True, description="Activates fortran support")
+    # Install-time footprint is dominated by ~8 k tutorial/example files.
+    # Give packagers a switch to trim them away (‘spack install petsc ~examples’)
```

Comment:
looks like `spack install petsc ~install-examples` is a bit overloaded. So I guess `examples`  is more appropriate.

Sorry about going back and forth on this :(